packages/core/lib/dlc/TxBuilder.ts

```diff
@@ -90,11 +90,11 @@ export class DlcTxBuilder {
     tx.addOutput(Value.fromSats(Number(fundingValue)), witScript);
     tx.addOutput(
       Value.fromSats(Number(offerChangeValue)),
-      Script.parse(StreamReader.fromBuffer(this.dlcOffer.changeSPK)),
+      Script.p2wpkhLock(this.dlcOffer.changeSPK.slice(2)),
```

Comment:
This PR also fixes a small issue with `DlcTxBuilder` where the generated funding tx had non-standard output SPKs